### DISCUSSION TITLE
feature: add @IsTime

### DISCUSSION THREAD

**Discussion by kimsey0** [typestack/class-validator#2364] [Feature Ideas]
### Description
It's currently not possible to validate time values like `12:34` using class-validator. These may come the [time input HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/time).

### Proposed solution
Add a new `@IsTime` validator which calls the `isTime` function from validator.js.

**Comment by kimsey0** [comment]
I just now noticed there's `@IsMilitaryTime`. 🤦 (The believe the term "military time" is basically unknown outside the United States.)

I'd still suggest an `@IsTime` that just allows passing in the options for the validator.js `isTime`.

**Comment by braaar** [comment]
Perhaps it would be a good idea to start using `isTime` for `@IsMilitaryTime` and add `@IsTime` as an alias for it. PR welcome.

**Comment by braaar** [comment]
I'm moving this to the discussion tab, as we are trying to keep the issue count low. Please see [this issue](https://github.com/typestack/class-validator/issues/1775) for an explanation.
